src/idiomorph.js

```diff
@@ -713,11 +716,19 @@ var Idiomorph = (function () {
                 ignoreActiveValue: mergedConfig.ignoreActiveValue,
                 idMap: createIdMap(oldNode, newContent),
                 deadIds: new Set(),
+                pantry: mergedConfig.twoPass && createPantry(),
                 callbacks: mergedConfig.callbacks,
                 head: mergedConfig.head
             }
         }
 
+        function createPantry() {
+            const pantry = document.createElement("div");
```

Comment:
maybe add `style='display:none'` to hide?

Reply:
also maybe throw an id on it?  `idiomorph-pantry`?

Reply:
IDK, `el.hidden = true` seems to be a higher level way to express the same thing as `style="display: none"`, and is just as good in terms of browser support  https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden#browser_compatibility

I'm not married to it, though. If you prefer the latter that's fine.

Reply:
> also maybe throw an id on it? `idiomorph-pantry`?

I actually had this at first, but ultimately decided to remove it, thinking that it could get weird with id collision if we have more than one instance of Idiomorph running on a page simultaneously, which seems plausible between Turbo and htmx. I'm not coming up with anything it buys us either, aside from someone maybe being able to pick it out visually in the dev tools html inspector while debugging. What do you think?